src/main/app/forms/validation/validators/minAmountRange.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+@ValidatorConstraint({ name: 'minAmountRange' })
+export class MinAmountRangeConstraint implements ValidatorConstraintInterface {
+
+  validate (value: any, args: ValidationArguments) {
+
+    const min: number = args.constraints[0]
+    const cannotStateInput = (args.object as any)[args.constraints[1]]
+
+    if (cannotStateInput === 'cannot' && (value === null || value === undefined)) {
```

Comment:
I can see you always check if value is null / undefined. Is there a real scenario that this var can have these values?